File: HW0Q5.m


pat1 = "autoNomy";
pat2 = "Exit";
for i = [0:2]
    inp = input("Enter the name of the course: ", "s");
    if contains(inp ,pat1,'IgnoreCase',true)
        disp('Correct')
        return
    elseif contains(inp ,pat2,'IgnoreCase',true)
        return 
    end 
end 

%https://stackoverflow.com/questions/38723138/matlab-execute-script-from-linux-command-line

File: HW0Q5.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Sat Aug 31 07:03:16 2024

@author: erik
"""
import re 
pattern1 = re.compile('autonomy', re.IGNORECASE)
pattern2 = re.compile('exit', re.IGNORECASE)
for i in range(3):
    ans = input("Please enter the title of this course:")

    if pattern1.search(ans):
        print('Correct')
        exit() 
    elif pattern2.search(ans):
        exit()
    

    